examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
+    {
       "name": "VOLUME_CAPACITY",
       "displayName": "Volume Capacity",
-      "description": "Volume space available for data, e.g. 512Mi, 2Gi.",
+      "description": "Volume space available for data, e.g. 500Mi, 2Gi.",
       "value": "1Gi",
       "required": true
+    },
+    {
+      "name": "JENKINS_IS_NAMESPACE",
+      "displayName": "Jenkins ImageStream Namespace",
+      "description": "The OpenShift Namespace where the Jenkins ImageStream resides.",
+      "value": "openshift"
+    },
+    {
+      "name": "JENKINS_IST",
+      "displayName": "Jenkins ImageStreamTag",
+      "description": "Tag (name) of the ImageStream in that namespace.",
```

Comment:
"Name of the ImageStreamTag to be used for the Jenkins image."

and the parameter name should be JENKINS_IMAGE_STREAM_TAG, users will be confused by "IST"